diff --git a/fesm2022/core.mjs b/fesm2022/core.mjs
index 83f747bc8abdaa442716af2fae24561ad134b952..c494c88264cb9dd0afab37d1dc60cefb65103cee 100755
--- a/fesm2022/core.mjs
+++ b/fesm2022/core.mjs
@@ -6915,7 +6915,9 @@ function symbolIterator() {
  * @publicApi
  */
 class QueryList {
-    static { Symbol.iterator; }
+    // Terser is not friendly with static initialization blocks:
+    // https://github.com/TrySound/rollup-plugin-terser/issues/112
+    // static { Symbol.iterator; }
     /**
      * Returns `Observable` of `QueryList` notifying the subscriber of changes.
      */
