diff --git a/node_modules/@angular/core/fesm2022/core.mjs b/node_modules/@angular/core/fesm2022/core.mjs
index 30b1cad..a54b3bf 100755
--- a/node_modules/@angular/core/fesm2022/core.mjs
+++ b/node_modules/@angular/core/fesm2022/core.mjs
@@ -22737,7 +22737,9 @@ function symbolIterator() {
  * @publicApi
  */
 class QueryList {
-    static { Symbol.iterator; }
+    // Terser is not friendly with static initialization blocks:
+    // https://github.com/TrySound/rollup-plugin-terser/issues/112
+    // static { Symbol.iterator; }
     /**
      * Returns `Observable` of `QueryList` notifying the subscriber of changes.
      */
